stonesoup/models/transition/linear.py

```diff
@@ -307,12 +308,92 @@ def covar(self, time_interval, **kwargs):
         return CovarianceMatrix(covar)
 
 
-class ConstantVelocity(LinearGaussianTransitionModel, TimeVariantModel):
+class ConstantNthDerivative(LinearGaussianTransitionModel, TimeVariantModel):
+    r"""Model based on the Nth derivative of position being constant,
```

Comment:
An alternative to the word "position" may be required. I assume that the same model can also be applied to variables like bearing, orientation, etc.